iphone/Classes/TiUIImageView.m

```diff
@@ -683,23 +697,15 @@ - (void)setImage_:(id)arg
     return;
   }
 
-  [imageview setImage:image];
+  [self setTintedImage:image];
+
   [(TiViewProxy *)[self proxy] contentsWillChange]; // Have to resize the proxy view to fit new subview size, if necessary
 
   if (currentImage != image) {
     [self fireLoadEventWithState:@"image"];
   }
 }
 
-- (void)setTintColor_:(id)value
```

Comment:
@hansemannn This code should not be removed. If you want set tintColor on rendered image, without this it is not possible. See following test case- 
`var win = Ti.UI.createWindow({
    backgroundColor: '#fff'
});
 
var btn = Ti.UI.createButton({
    title: 'Load remote image'
});
 
var image = Ti.UI.createImageView({
    top: 60,
    width: 90,
    height: 90,
    tintColor: 'red',
    image: Ti.UI.createView({ backgroundColor: 'gray', width: 270, height: 270 }).toImage()
});
 
btn.addEventListener('click', function () {
    var url = 'https://ss3.4sqi.net/img/categories_v2/food/breakfast_90.png';
 
    var client = Ti.Network.createHTTPClient({
        onload: function () {
            const blob = client.responseData;
            const cachedFile = Ti.Filesystem.getFile(Ti.Filesystem.applicationCacheDirectory, 'icon.png');
            cachedFile.write(blob);
 
            image.image = cachedFile.nativePath;
        },
        onerror: function (e) {
            Ti.API.error(e.error);
        }
    });
 
    client.open('GET', url);
    client.send();
});
 
win.add([ btn, image ]);

var btn2 = Ti.UI.createButton({
    title: 'Set green tint color to image',
    bottom: 100
});

btn2.addEventListener('click', function () {
image.tintColor = 'green'
});

win.add(btn2);

win.open();`

Run the test app. Click on 'Set green tint color to image'. It will not apply. Ideally green color should be applied on image.